### PR TITLE
Find max recursion

### PR DESCRIPTION
**Pull request by Exortions**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/FindMaxRecursion.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  if (len === 0 || !arr) return undefined
+
+  if (left >= len || left < -len || right >= len || right < -len)
+    throw new Error('Index out of range')
+
+  if (left === right) return arr[left]
+
+  // left + right shifted is the middle index
+  const mid = (left + right) >> 1
```
Comment (appgurueu) [inline Maths/FindMaxRecursion.js:28]: Consider explaining that `n >> m` is `floor(n / pow(2, m))` and in this case `floor(n / 2)`

**Review by appgurueu — APPROVED** [review]

**Review by appgurueu — CHANGES_REQUESTED** [review]
```
standard: Use JavaScript Standard Style (https://standardjs.com/)
standard: Run `standard --fix` to automatically fix some problems.
  /home/runner/work/JavaScript/JavaScript/Maths/FindMaxRecursion.js:17:26: Missing space before function parentheses.
  /home/runner/work/JavaScript/JavaScript/Maths/FindMaxRecursion.js:22:3: Expected { after 'if' condition.
```
Please fix the style.

File: Maths/FindMaxRecursion.js (hunk 1 above)
Comment (Exortions) [reply Maths/FindMaxRecursion.js:28]: Yep, will do

**Comment by Exortions** [comment]
> ```
> standard: Use JavaScript Standard Style (https://standardjs.com/)
> standard: Run `standard --fix` to automatically fix some problems.
>   /home/runner/work/JavaScript/JavaScript/Maths/FindMaxRecursion.js:17:26: Missing space before function parentheses.
>   /home/runner/work/JavaScript/JavaScript/Maths/FindMaxRecursion.js:22:3: Expected { after 'if' condition.
> ```
> 
> Please fix the style.

Yeah, when I committed the style was fine according to standard, but I'll fix.

**Pull request closed without merging**